obs_status.yml

```diff
@@ -0,0 +1,2147 @@
+  status: 1
+- agasc_id:
+  - 991560472
+  comments: BSH at start of observation
+  mp_starcat_time: 2017:068:16:27:18.823
+  status: 1
+- agasc_id:
+  - 991563000
+  comments: BSH at start of observation
+  mp_starcat_time: 2017:068:16:27:18.823
+  status: 1
+- agasc_id:
+  - 991564392
+  comments: BSH at start of observation
+  mp_starcat_time: 2017:068:16:27:18.823
+  status: 1
+- agasc_id:
+  - 798238560
+  comments: Tracking affected by imposter
+  mp_starcat_time: 2017:075:03:00:23.891
+  status: 1
+- agasc_id:
+  - 1200756680
+  comments: NSM at start of observation
+  mp_starcat_time: 2017:090:18:27:10.378
+  status: 1
+- agasc_id:
+  - 1200757192
+  comments: NSM at start of observation
+  mp_starcat_time: 2017:090:18:27:10.378
+  status: 1
+- agasc_id:
+  - 1200758760
+  comments: NSM at start of observation
+  mp_starcat_time: 2017:090:18:27:10.378
+  status: 1
+- agasc_id:
+  - 1200760024
+  comments: NSM at start of observation
+  mp_starcat_time: 2017:090:18:27:10.378
+  status: 1
+- agasc_id:
+  - 1200888464
+  comments: NSM at start of observation
+  mp_starcat_time: 2017:090:18:27:10.378
+  status: 1
+- agasc_id:
+  - 45616472
+  comments: Faint star affected by imposter. Set mag to MAXMAG
+  mp_starcat_time: 2017:119:02:36:19.023
+  status: 1
+- agasc_id:
+  - 311824376
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2017:198:09:20:24.644
+  status: 1
+- agasc_id:
+  - 913441888
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2017:259:02:11:04.769
+  status: 1
+- agasc_id:
+  - 377102496
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2017:314:20:46:46.000
+  status: 1
+- agasc_id:
+  - 92812824
+  comments: Almost never tracked. Set mag to MAXMAG
+  mp_starcat_time: 2017:354:04:35:24.949
+  status: 1
+- agasc_id:
+  - 653529952
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2018:002:23:37:48.551
+  status: 1
+- agasc_id:
+  - 123874048
+  comments: Bright star
+  mp_starcat_time: 2018:014:04:03:16.301
+  status: 1
+- agasc_id:
+  - 797847184
+  comments: Tracking affected by imposter. Set mag to 12.3 +- 0.2
+  mp_starcat_time: 2018:068:11:54:25.784
+  status: 1
+- agasc_id:
+  - 1105068512
+  comments: spoiler 22 arcsec away
+  mp_starcat_time: 2018:069:03:04:54.838
+  status: 1
+- agasc_id:
+  - 872547280
+  comments: Large mag variation. Star is a known eclipsing binary
+  mp_starcat_time: 2018:074:18:43:05.432
+  status: 1
+- agasc_id:
+  - 651037128
+  comments: Almost never acquired. Faint star affected by imposters. Marginally acceptable
+  mp_starcat_time: 2018:081:21:42:04.564
+  status: 0
+- agasc_id:
+  - 899950912
+  comments: Faint star. Affected by hot pixels. Set mag to MAXMAG
+  mp_starcat_time: 2018:155:22:59:34.754
+  status: 1
+- agasc_id:
+  - 738075032
+  comments: Spoiler 30 arcsec away
+  mp_starcat_time: 2018:158:10:17:36.762
+  status: 1
+- agasc_id:
+  - 49818760
+  comments: Faint star. Affected by imposter. Set mag to MAXMAG
```

Comment:
For a couple of these, I'm not sure if we win much setting to MAXMAG over the observed mag?  This one, bad pixels or not, looks like it has 29% tracked and observed mag 11.03.   Maxmag = 10.859.  But fine this way too.

Reply:
I agree in general (I have not gone to see this specific observation now). Observations that are borderline might have gone one way or another depending on the mood :-)

The question is whether we need to make a better guideline to decide.